packages/cli/test/fixtures/unit/commands/deploy/node-low-starting-range/api/es6.js

```diff
@@ -0,0 +1 @@
+export default (req, res) => res.end('Vercel');
```

Comment:
Not a huge deal, but we could remove some of these API functions. Just keep the `index.js` one.